template-compose/app/build.gradle.kts

```diff
@@ -11,6 +13,13 @@ plugins {
 }
 
 val keystoreProperties = rootDir.loadGradleProperties("signing.properties")
+val getVersionCode: () -> Int = {
+    if (project.hasProperty("versionCode")) {
+        (project.property("versionCode") as String).toInt()
+    } else {
+        Versions.ANDROID_VERSION_CODE
+    }
+}
```

Comment:
can we optimize this code to eliminate the `if`?

```suggestion
val getVersionCode: () -> Int = {
    (project.property("versionCode") as? String)?.toInt() ?: Versions.ANDROID_VERSION_CODE
}
```

Reply:
@luongvo It doesn't work. 😅 
![SCR-25660718-njws](https://github.com/nimblehq/android-templates/assets/12026942/dbf24ec0-b302-4a2c-b727-713e50187038)